peripherals/rcu/rcu_e1.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 The gd32-rs authors.
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+_include:
+  - rcu_common.yaml
+
+RCU:
+  CTL0:
+    "PLL?EN":
+      Disabled: [0, "PLL Disabled"]
+      Enabled: [1, "PLL Enabled"]
+    "PLL?STB":
+      _read:
+        NotReady: [0, "PLL is not stable"]
+        Ready: [1, "PLL is stable"]
```

Comment:
Any particular reason for breaking these out to a separate file, versus making `rcu_common.yaml` more generic for `PLL*STBIC` and so on? They seem pretty similar, I'd be inclined to take the latter approach for these too (i.e. change `rcu_common.yaml` to have `PLL*STB` and `PLL*EN`, if that works).

Reply:
Oh, I didn't see that! That's a great idea :)
Should I keep the `Off`/`On` enumerated values? I used `Disabled`/`Enabled` because it felt more common but it would introduce a breaking change in the API

Reply:
I'd prefer to keep the existing names if that's alright, just for consistency and to avoid a breaking change. FWIW the STM32 PAC crates seem to use `On`/`Off` for these so I copied that.

Reply:
No problem, I just merged the fields in the last commit